Cargo.toml

```diff
@@ -11,22 +11,22 @@ edition = "2021"
 rust-version = "1.67.0"
 
 [features]
-default = []
-cli = ["dep:clap"]
+default = ["std"]
+std = []
+cli = ["std", "dep:clap"]
```

Comment:
It just occurred to me, that we don't need the `std` feature.

Nothing in this change requires `libstd` as everything is in `libcore`. So the lib can be `nostd`, and anyone can import it into std or nostd projects.

Reply:
Yes, I will modify it after the merge of #70.